docs(color): DLT-2480 DLT-2446 improve color usage guidelines (#808)
dialpad/dialtone: apps/dialtone-documentation/docs/design/colors/usage/index.md

---
title: Usage
description: A systematic and accessible color palette that supports both functional needs and personalized experiences within our products.
---

<div class="d-m32"></div>

## Overview

Dialtone's color system offers a consistent visual foundation with a brand purple, versatile grayscale, and semantic status colors. Subtle gradients are used for promotional accents.

What we prioritize is semantic clarity, ensuring each color serves a distinct purpose within the UI for clear communication and consistency...

<div class="d-m32"></div>

> <div class="d-bgc-purple-400 d-bgo10 d-bar12 ">

Reviewer: ```suggestion
<div class="d-bgc-brand d-bar8">
```

And repeat that for all others.

 <svg-loader class="d-fl1" name="color-table" />

>  </div>

Reviewer: `d-fl1`?

### Brand Colors

  Brand colors are non-negotiable for immediate product recognition. Each Dialpad product is visually defined by its unique brand color.

### Neutral Colors

Blacks, whites, and grays—form the bedrock of our interfaces. They dictate surfaces, text, and layout, often signaling state changes within components. Lighter neutrals strategically guide the eye, establishing a clear visual hierarchy

### Semantic Colors

> Semantic colors are our messengers, conveying feedback, status, and urgency. They are fundamental to Dialtone's token system, ensuring that critical information is always visually prioritized.

Reviewer: I'm not sure this is adding any value. A few options: rewrite, or move their lines closer to the associated color they address, or perhaps remove and revisit later.

<div class="d-m96"></div>

> ---

Reviewer: Remove

Reviewer: And remove the others one that do the same.

## Use Semantic Color Tokens

Dialtone uses **semantic tokens** for color, representing a color's *purpose* in the UI. See our [See our design token list](https://dialtone.dialpad.com/tokens/) for descriptions guiding their application.

<div class="d-bgc-purple-400 d-bgo10 d-bar12"><svg-loader class="d-fl1" name="color-tokens" /></div>

> <div class="d-m32"></div>

Reviewer: Remove 32 margin. Just let the content flow and let the hierarchy do its thing.

Reviewer: And remove the others one that do the same.

It's crucial to understand that while these semantic tokens are built upon our underlying **base color palette**, the work of selecting the appropriate base color for each specific UI context has already been meticulously done for you.

 <div class="d-bgc-purple-400 d-bgo10 d-bar12"><svg-loader class="d-fl1" name="color-semantic" /></div>

<div class="d-m96"></div>

---

## Color roles

Color roles describe the purpose of how color is used. Each uses neutrals for general UI and status colors for impact.

 <div class="d-bgc-purple-400 d-bgo10 d-bar12"><svg-loader class="d-fl1" name="color-roles" /></div>

We categorize color application by semantic roles such as: **text, surfaces, borders, and themes**. Each uses neutrals for general UI and status colors for impact.

<div class="d-m48"></div>

> ### Text Colors

Reviewer: ```suggestion
### Foreground Colors
```

> Text colors are the visual language of our content, applied across all text and type elements. You can utilize foreground tokens which can be found within the [tokens documentation here](https://dialtone.dialpad.com/tokens/).

Reviewer: ```suggestion
Foreground colors are the visual language of our content, applied across all content considered to be "in the foreground", i.e. text and icons. You can utilize foreground tokens which can be found within the [tokens documentation here](https://dialtone.dialpad.com/tokens/).
```

<div class="d-d-grid d-g24 d-g-cols2">
  <div>
    <div class="d-bgc-purple-400 d-bgo10 d-bar12"><svg-loader class="d-fl1" name="color-text-f" /></div>
    <h4>Foreground Text</h4>
    Neutral text colors form the backbone of our content, used for headers, body text, forms, and more.
  </div>
  <div>
     <div class="d-bgc-purple-400 d-bgo10 d-bar12"><svg-loader class="d-fl1" name="color-text-s" /></div>
    <h4>Status Text</h4>
    Status text colors are critical for highlighting key information:
    <ul>
      <li><em>Red:</em> Signals critical attention.</li>
      <li><em>Yellow:</em> Indicates warnings.</li>
      <li><em>Green:</em> Denotes positive actions.</li>
    </ul>
  </div>

> </div>

Reviewer: Need to have icons represented in here as using foreground colors.

</template>

<template #dont>

<div class="d-bgc-purple-400 d-bgo10 d-bar12"><svg-loader class="d-fl1" name="color-contrast-dont" /></div>

</template>

</dialtone-usage>

<div class="d-m48"></div>

### Uniformity

Maintain color consistency across similar components to build intuitive user patterns.

<dialtone-usage>
<template #do>

<div class="d-bgc-purple-400 d-bgo10 d-bar12"><svg-loader class="d-fl1" name="color-uniform-do" /></div>
</template>

<template #dont>

<div class="d-bgc-purple-400 d-bgo10 d-bar12"><svg-loader class="d-fl1" name="color-uniform-dont" /></div>

</template>

</dialtone-usage>

<div class="d-m96"></div>

---

## Modes

Dialtone is designed to seamlessly support both light and dark themes. We supply colors through semantic tokens that automatically adapt their underlying values depending on the active mode, ensuring consistent meaning and optimal contrast in any environment.

<div class="d-bgc-purple-400 d-bgo10 ">
 <svg-loader class="d-fl1" name="color-mode" />
 </div>

<div class="d-m96"></div>

---

## Related

- If you need full list of our color palette, [Check out our list of colors](https://dialtone.dialpad.com/design/colors/).

> - [See our entire token catalog with full descriptions and values of all tokens.](https://dialtone.dialpad.com/tokens/).

Reviewer: Use relative paths.
